### DISCUSSION TITLE
World’s First Social Media Smart App Inclusive Productivity Focused

### DISCUSSION THREAD

**Discussion by ElioRivas** [XRPLBounties/Proposals#35] [Ideas]
The Next Generation of Smart Mobile Apps for Inclusive Social Media
Innovating for a global need!

**What's the Problem?**
Latest statistics about deafness and hearing loss in the world, 1.5 billion people live with some degree of hearing loss, of which
about 430 million need rehabilitation services. By 2050 it is projected that there will be almost 2.5 bi llion people with some
degree of hearing loss and that at least 700 million will require rehabilitation ( In the US nearly 40 million people are
struggling with the same problem. 

According to the 5 Meeting Industry Trends That Are Changing the Face of Events ( 1) millennials are the decision makers
now, 2) Attendees want more control over the meeting agenda, 3) Meetings should be purposeful experiences, 4) Technology
helps planners pull off more than ever before and 5) Where you meet matters more than ever before Current
social media applications stopped continue innovating due lack of vision for integrating people that have different
difficulties communicating.

1.https://www.who.int/en/news-room/fact-sheets/detail/deafness-and-hearing-loss 
2. https://www.socialtables.com/blog/event-planning/meeting-industry-trends/

**Proposed solution**

Developing and launch to market a mobile application (Android, iOS & Web) capable of:

1) Making a FaceTime instant translation from sign language to voice and vice versa, between people who use sign language to
communicate and people who do not know. For this, it will use Neural networks and Machine Learning.

2) The mobile app will automatically create meeting reminders based on user interaction via text messaging, voice, and video calls (in the Future). This will be carried out based on text patterns like that usually users text each other e.g.: see you today at 7 pm (it will use Text Mining).

3) The application also will have the same features tha t current messaging mobile apps like WhatsApp, Telegram, Signal, etc.

**Value**
With the Mobile Application providing these two new features, besides existing features, that current messaging apps do not provide in just one application, we expect to be able to break rapidly into the Social Media and Text Messaging Applications market.

> **These two new features such as:**

> 1) For deaf mute users we will provide them a tool that contributes suppress the language barrier, they will be able to interact more in society suppressing discrimination due to their handicap, they will continue growi ng on their own goals and get better opportunities in all respect, we will contribute in creating a more inclusive society with the technology.

> Note: Cochlear implants cost over US$ 3,500.00 with the Application the user will save money by reducing the cos t of communication.

> 2) For the automatic reminder creating feature we will contribute to enhancing the meetings interaction and punctuality and increase people and companies efficiency. Users will not need to count with a handbook agenda instead of an App.

**Final thoughts next steps**
This new cutting edge application not only will allow us to break into the social media and text messaging market but it will also expand our brand in a positive way focusing on innovation as the key to contributing to society.

XRPL will be seen as the first organization capable and interested in creating cutting edge technology with an inclusive approach in the social media market.

In future works, we will continue improving the application so that no matter the language, we will provide the capability to do instant translations in real time. For instance, there are two users and one speaks English and the other Spanish: when the user that is speaking English starts using the app, it automatically translates in voice to Spanish to the user that only speaks Spanish and vice versa.
[Executive Summary-Inclusive Mobile App.pdf](https://github.com/XRPLBounties/Proposals/files/9482815/Executive.Summary-Inclusive.Mobile.App.pdf)

**Comment by JST5000** [comment]
Hey ElioRivas, just wanted to let you know that if this is a project you're personally pursuing and you want to see get funding for the idea, it's probably best to apply to the Grants program as opposed to proposing it as a bounty here. Bounties are more for work you want other people to do or for projects which have a very clear scope.
